src/components/ComponentConfiguration.stories.tsx

```diff
@@ -1691,3 +1691,59 @@ export const BSN: Story = {
     expect(args.onSubmit).toHaveBeenCalled();
   },
 };
+
+export const Address: Story = {
+  render: Template,
+  name: 'type: address',
+
+  args: {
+    component: {
+      id: 'wekruya',
+      type: 'address',
+      key: 'address',
+      label: 'An address',
+      validate: {
+        required: false,
+      },
+    },
+  },
+
+  play: async ({canvasElement, args}) => {
+    const canvas = within(canvasElement);
+
+    await expect(canvas.getByLabelText('Label')).toHaveValue('An address');
+    await waitFor(async () => {
+      await expect(canvas.getByLabelText('Property Name')).toHaveValue('anAddress');
+    });
+    await expect(canvas.getByLabelText('Description')).toHaveValue('');
+    await expect(canvas.getByLabelText('Tooltip')).toHaveValue('');
+    await expect(canvas.getByLabelText('Show in summary')).toBeChecked();
+    await expect(canvas.getByLabelText('Show in email')).not.toBeChecked();
+    await expect(canvas.getByLabelText('Show in PDF')).toBeChecked();
+
+    // ensure that changing fields in the edit form properly update the preview
+    const preview = within(canvas.getByTestId('componentPreview'));
+
+    await userEvent.clear(canvas.getByLabelText('Label'));
+    await userEvent.type(canvas.getByLabelText('Label'), 'Updated preview label');
+    expect(await preview.findByText('Updated preview label'));
+
+    const previewInput = preview.getByText('Updated preview label');
+    await expect(previewInput).toHaveDisplayValue('');
```

Comment:
```suggestion
    await expect(previewInput).toBeVisible();
```

it's not an input field